docs/source/ops/system_architecture_faq.rst

```diff
@@ -0,0 +1,89 @@
+
+**Further Reading**:
+
+- `Transaction processing in Canton <../canton/architecture/overview.html#transaction-processing-in-canton>`__
+- :ref:`execution_model`
+
+How does Canton ensure privacy?
+///////////////////////////////
+
+Most sequencer backend options have only limited privacy features. In order to provide privacy even against the operator of the sequencers and sequencer backend, Canton adds its own privacy by encrypting all message payloads sent via the sequencer to be readable only by the intended recipients. That includes the transaction payloads sent as part of confirmation requests.
+
+Canton messages are multi-cast, meaning they can have multiple recipients, and in some cases, like commit requests, have different views for the different recipients. The submitter of a message generates a single-use symmetric View Encryption Key for each view, and encrypts the views using those. It then encrypts only a seed for that View Encryption Key using the public half of an assymmetric Participant Encryption Keys that each Canton node publishes.
+
+The View Encryption Keys are kept - encrypted for each receiver - with the message payload itself. A receiving node uses their Participant Encryption key to decrypt the seed of the View Encryption Key for each of the views they are entitled to read, and uses a key derivation function (HKDF to be precise) to recover the View Encryption Key and read the view. 
+
+The supported encryption algorithms for asymmetric encryption (Participant Encryption Keys) and symmetric encryption (View Encryption Keys) are listed in the documentation `here <../canton/usermanual/security.html#cryptographic-key-usage>`__.
+
+**Further Reading:**
+
+- `Encryption Keys <../canton/usermanual/security.html#participant-encryption-key>`__
+- `View Encryption Keys <../canton/usermanual/security.html#view-encryption-key>`__
+- `Cryptographic Key Usage <../canton/usermanual/security.html#cryptographic-key-usage>`__
+
+.. _where_holden_source:
+
+Where does "the golden source" of Daml Ledger data live in Canton?
+//////////////////////////////////////////////////////////////////
+
+The short answer is that Daml Ledger data lives both on the Canton participant nodes, and on the squencer backend, meaning the blockchain or database enabled by the driver. The data is stored in the two places in different ways, but fully consistent thanks ot Canton's deterministic execution model.
+
+All communication between Canton nodes, including the confirmation requests for transactions and the resulting confirmations and rejections, are stored on sequencer backend. Since Daml and Canton are built around deterministic execution, you can thus consider that data on the sequencer backend, together with the Participant Encryption Keys, to be a complete copy of the Daml Ledger. 
```

Comment:
```suggestion
All communication between Canton nodes, including the confirmation requests for transactions and the resulting confirmations and rejections, are stored on the sequencer backend. Since Daml and Canton are built around deterministic execution, you can thus consider that data on the sequencer backend, together with the Participant Encryption Keys, to be a complete copy of the Daml Ledger. 
```